apps/daimo-mobile/src/view/screen/send/SendScreen.tsx

```diff
@@ -50,123 +27,75 @@ export default function SendScreen({ route }: Props) {
-  const [noteCreated, setNoteCreated] = useState(false);
-  const onNoteCreated = useCallback(() => setNoteCreated(true), []);
+function SendNav() {
+  // Navigation
+  const [tab, setTab] = useState<SendTab>("Search");
+  const [tabs] = useState(["Search", "Send Link", "Scan"] as SendTab[]);
+  const search = useCallback(() => setTab("Search"), []);
 
   return (
-    <View style={ss.container.fullWidthSinglePage}>
-      <Header />
-      <ScrollView
-        contentContainerStyle={styles.vertStretch}
-        bounces={false}
-        keyboardShouldPersistTaps="handled"
-      >
-        <KeyboardAvoidingView behavior="position" keyboardVerticalOffset={128}>
-          {recipient && (
-            <SetAmount recipient={recipient} dollars={dollars || 0} />
-          )}
-          {!recipient && (
-            <>
-              {tab !== "createNote" && (
-                <SegmentedControlFixed
-                  values={tabs}
-                  selectedIndex={tab === "scan" ? 1 : 0}
-                  onValueChange={setSegmentVal}
-                  fontStyle={{ fontSize: 16 }}
-                  activeFontStyle={{ fontSize: 16 }}
-                  style={{ height: 40 }}
-                />
-              )}
-              {tab !== "createNote" && <Spacer h={16} />}
-              {tab === "search" && <SearchTab />}
-              {tab === "scan" && <ScanTab hide={search} />}
-              {tab === "createNote" && (
-                <SendAmountChooser
-                  actionDesc={
-                    noteCreated
-                      ? "Payment link created"
-                      : "Creating payment link"
-                  }
-                  dollars={noteDollars}
-                  onSetDollars={noteCreated ? undefined : setNoteDollars}
-                  onCancel={noteCreated ? undefined : search}
-                />
-              )}
-            </>
-          )}
-        </KeyboardAvoidingView>
-      </ScrollView>
-      {!recipient && tab === "search" && (
-        <ButtonWithStatus
-          button={
-            <ButtonBig
-              type="primary"
-              title="Create Payment Link"
-              onPress={createNote}
-            />
-          }
-          status={sendViaAppStr}
-        />
-      )}
-      {!recipient && tab === "createNote" && (
-        <SendNoteButton dollars={noteDollars} onCreated={onNoteCreated} />
-      )}
-      {recipient && (
-        <SendButton
-          recipient={recipient}
-          dollars={dollars || 0}
-          requestId={requestId}
-        />
-      )}
+    <View>
+      <SegmentedControlFixed
+        values={tabs}
+        selectedIndex={tabs.indexOf(tab)}
+        onValueChange={setTab}
+        fontStyle={{ fontSize: 16 }}
+        activeFontStyle={{ fontSize: 16 }}
+        style={{ height: 40 }}
+      />
+      <Spacer h={8} />
+      {tab === "Search" && <SearchTab />}
+      {tab === "Send Link" && <SendNoteTab />}
```

Comment:
should there be a `hide` or similar param here too like scan?

Reply:
fixed, removed from ScanTab